### PR TITLE
PETL-89: Add Conditional to SqlServerImportJob

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: src/main/java/org/pih/petl/job/datasource/DatabaseUtil.java [hunk 1]
```diff
@@ -44,4 +44,18 @@ public static Integer rowCount(Connection c, String table) throws SQLException {
         String query = "select count(*) from " + table;
         return qr.query(c, query, new ScalarHandler<>());
     }
+
```
Comment (mogoodrich) [inline src/main/java/org/pih/petl/job/datasource/DatabaseUtil.java:47]: These two methods are just used by new test cases, see below.

File: src/test/java/org/pih/petl/job/SqlServerImportJobTest.java [hunk 2]
```diff
@@ -28,12 +29,35 @@
         SpringRunnerTest.setupEnvironment();
     }
 
+    @After
+    public void dropTablesInTargetDB() throws Exception{
```
Comment (mogoodrich) [inline src/test/java/org/pih/petl/job/SqlServerImportJobTest.java:33]: I tried to fiddle around with dirtying the context, etc, but couldn't get that to work, so just wrote this to drop the target table after each run.

File: src/test/java/org/pih/petl/job/SqlServerImportJobTest.java [hunk 3]
```diff
@@ -28,12 +29,53 @@
...
+        verifyRowCount("encounter_types", 62);
+
+        // by default, table should be dropped and recreated on each run, so consecutive runs should return the same result
+        etlService.executeJob("sqlserverimport/job.yml");
+        verifyTableExists("encounter_types");
+        verifyRowCount("encounter_types", 62);
+    }
+
+
+    @Test
```
Comment (mogoodrich) [inline src/test/java/org/pih/petl/job/SqlServerImportJobTest.java:54]: Added this test for the dropAndRecreateTable functionality I added last week (also changed the above test to confirm that is *does* drop and recreate by default)

**Review by cioan — APPROVED** [review]
Looks good, @mogoodrich ! THanks!

**Review by mseaton — APPROVED** [review]
Great job, looks good to me

**Comment by mogoodrich** [comment]
Thanks @mseaton  @cioan

**Pull request merged**